core/src/test/java/org/lflang/tests/compiler/LinguaFrancaValidationTest.java

```diff
@@ -1399,29 +1411,34 @@ private Model createModel(TargetProperty key, String value) throws Exception {
   }
 
   /** Perform checks on target properties. */
-  @Test
-  public void checkTargetProperties() throws Exception {
+  @TestFactory
+  public Collection<DynamicTest> checkTargetProperties() throws Exception {
+    List<DynamicTest> result = new ArrayList<>();
+
     for (TargetProperty prop : TargetProperty.getOptions()) {
       if (prop == TargetProperty.CARGO_DEPENDENCIES) {
         // we test that separately as it has better error messages
-        return;
+        continue;
```

Comment:
Notice this change: I believe some properties were not being tested because we used to exit early. Fixing this revealed that there were no checks being executed for properties which have type "file", and further, that the validator does not do any check for those properties, so there is nothing to test for those properties. I filtered out tests for properties of this type as a workaround, although maybe the better fix is to add a validator check. The thing is I don't know how that would be written because it would have to deal with relative paths.

Reply:
Thanks for the heads up. I created an issue for this...